Makefile

```diff
@@ -69,15 +83,23 @@ all: install lint test
 
 build: go.sum
 ifeq ($(OS),Windows_NT)
-	go build -mod=readonly $(BUILD_FLAGS) -o build/wasmd.exe ./cmd/wasmd
-	go build -mod=readonly $(BUILD_FLAGS) -o build/wasmgovd.exe ./cmd/wasmgovd
+    # wasmd nodes not supported on windows, maybe the cli?
 	go build -mod=readonly $(BUILD_FLAGS) -o build/wasmcli.exe ./cmd/wasmcli
 else
 	go build -mod=readonly $(BUILD_FLAGS) -o build/wasmd ./cmd/wasmd
 	go build -mod=readonly $(BUILD_FLAGS) -o build/wasmgovd ./cmd/wasmgovd
 	go build -mod=readonly $(BUILD_FLAGS) -o build/wasmcli ./cmd/wasmcli
 endif
 
+build-coral: go.sum
+ifeq ($(OS),Windows_NT)
+    # wasmd nodes not supported on windows, maybe the cli?
+	go build -mod=readonly $(CORAL_BUILD_FLAGS) -o build/coral.exe ./cmd/wasmcli
+else
+	go build -mod=readonly $(CORAL_BUILD_FLAGS) -o build/corald ./cmd/wasmd
+	go build -mod=readonly $(CORAL_BUILD_FLAGS) -o build/coral ./cmd/wasmcli
```

Comment:
Note: I was expecting some `coralcli` like `gaiacli` or `wasmcli`

Reply:
I thought that, but the naming is pretty bad. What other unix commands use that naming?

`sshd` -> `ssh`
`mongod` -> `mongo`
`psqld` -> `psql`
...

I do see `wasm` as a poor command name, so I will leave it, but for testnets `coral`/`corald` seems nicer to use.

Reply:
I am a bit tired of following all the conventions of the sdk

Reply:
np. Just wanted to highlight this so that we can address this is the testnet doc.

Reply:
Good to make this clear (also the reasoning)

Reply:
Can I get an approval and merge? Or are there other issues?